docs/articles/new-expensify/billing-and-subscriptions/tax-exempt

```diff
@@ -0,0 +1,26 @@
+---
+title: Tax Exempt
+description: Tax-exempt status in Expensify for organizations recognized by the IRS or local tax authorities. 
+---
+
+# Overview
```

Comment:
It's a minor quibble but `#` is `<h1>` and it's for titles. Let's make this a subheading with `##`

```suggestion
## Overview
```